test/support/solr_test_support.ex

```diff
@@ -2,7 +2,14 @@ defmodule SolrTestSupport do
   def mock_solr_documents(count \\ 100) do
     for n <- 1..count do
       date = 2025 - n
-      page_count = Enum.random(1..10)
+      page_count = 7
```

Comment:
Can you add a comment about why page count is 7, and under what circumstances it could / should be changed? At some point we may want more flexibility here, maybe via a function param?